client/my-sites/hosting/support-card/style.scss

```diff
@@ -1,21 +0,0 @@
-.support-card {
-
-	&__avatars {
-		display: flex;
-		justify-content: space-between;
-		align-items: center;
-		flex-wrap: wrap;
-		height: 42px;
-		overflow: hidden;
-		margin-bottom: 16px;
-		margin-right: -8px;
-
-		.gravatar {
-			margin-right: 8px;
-		}
-	}
-
-	p {
-		margin-bottom: 16px;
```

Comment:
We should maintain this style rule for paragraphs, so all elements in the support card are separated by 16px.